ebuild-maintenance/text.xml

```diff
@@ -164,14 +164,16 @@ Linux Kernel patch guideline</uri>. Additionally, the following tags
 are used in Gentoo:
 
 <ul>
-<li><c>Gentoo-Bug:</c> Use this to reference bugs in Gentoo Bugzilla
-by either the bug ID or the bugzilla URI. Multiple bugs can be
-referenced by inserting more tags of this type or separating bug IDs
-with a comma in the same tag. The bug IDs may include an optional '#'
-prefix. There is no consensus on referencing bugs in the summary line
-versus referencing with a tag in the message body. The developer has
-the option to choose either.
-</li>
+<li><c>Bug:</c> Use this to reference bugs <e>without</e> closing them.
+The value needs to include a full URL to the bug, and can contain only
+a single URL. If a bug on Gentoo Bugzilla, or an issue or a pull request
+on GitHub is referenced, a reference to the commit will be added
+automatically.</li>
+<li><c>Closes:</c> Use this to reference bugs and close them
+automatically. Alike <c>Bug:</c>, the value needs to include a full URL
```

Comment:
"Full URL" may be misleading. We allow abbreviated https://bugs.gentoo.org/123456 but don't require the full https://bugs.gentoo.org/show_bug.cgi?id=123456 form, right?

Reply:
Just 'URL'?

Reply:
Fixed.